Translation/Extractor/File/DefaultPhpFileExtractor.php

```diff
@@ -140,7 +150,7 @@ public function enterNode(Node $node)
 
         $id = $node->args[0]->value->value;
 
-        $index = 'trans' === strtolower($node->name) ? 2 : 3;
+        $index = $this->methodsToExtractFrom[strtolower($node->name)];
         if (isset($node->args[$index])) {
```

Comment:
won't this result in an undefined exception if $node->name doesn't exist in methodsToExtractFrom ?